edtLHD/src/main/java/fr/univtln/lhd/facade/Schedule.java

```diff
@@ -105,6 +110,60 @@ public static List<Slot> getSchedule(Group group, Interval timerange) {
         return schedule.filterEverySlotOutsideTimerange(allSlotOfGroup, timerange);
     }
 
+    /**
+     * Get Student of Database via email & password
+     * @param email Email of the student
+     * @param password password of the student
+     * @return Student entity if exist in database, otherwise return null.
+     */
+    public static Student getStudentFromAuth(String email, String password){
+        StudentDAO dao = StudentDAO.getInstance();
+        Student student = null;
```

Comment:
I'd prefer an Optional here, this is a perfect example of a good usecase.

Reply:
i thought about it while writing it, and for me from the facade Schedule it's unecessary, the dao alreay returns an optional which will throw an exception if null
but fine.

Reply:
Let's just return that optional then, I believe the caller should handle the exception at that level.

Reply:
Here, I agree with @anastygnome